### PR TITLE
Feature/sleep class

### PR DESCRIPTION
**Pull request by st0nebreaker**
#### What's this PR do?
Refactors some for loops to other prototypes. Parses out users' name based off userID for later Sleep methods.

#### Where should the reviewer start?
files changed, local machine

#### Any background context you want to provide?
This broke a few npm tests, will fix later tonight

### CODE REVIEW

File: src/Sleep.js [hunk 1]
```diff
@@ -80,18 +82,21 @@ class Sleep {
...
 
 			const sortedSleepies = filterDate.sort((a, b) => b.hoursSlept - a.hoursSlept);
 			
-			//TODO: find user's name based off their ID
-			return `User #${sortedSleepies[0].userID} slept the most this day, they slept ${sortedSleepies[0].hoursSlept} hours -- WOW!`
+			const userName = givenClass.data.find(user => {
+				return user.id === sortedSleepies[0].userID;
+			}).name;
+
+			return `User ${userName} slept the most this day, they slept ${sortedSleepies[0].hoursSlept} hours -- WOW!`
```
Comment (st0nebreaker) [inline src/Sleep.js:96]: added another parameter for findSleepiest(), intakes a class too, later .find() on line 92 will find the user's name cycling over the UserRepo class to fetch the name

File: src/Sleep.js [hunk 2]
```diff
@@ -111,9 +116,14 @@ class Sleep {
...
-
+				
 				if (avg > 3) {
-					accu.push(Number(user));
+					let userName;
+					userName = givenClass.data.find(currentUser => {
+						return currentUser.id === Number(user);
+					});
+
+					accu.push(userName.name);
```
Comment (st0nebreaker) [inline src/Sleep.js:126]: same as before, iterates over the UserRepo class (done so in index.html file), matches the currentUser.id with the user # in this function. line 126 pushes the user's name into the array

File: src/scripts.js [hunk 3]
```diff
@@ -43,8 +43,8 @@ const updateHydration = (currentHydro) => {
 	const avgOunces = currentHydro.findHydrationAverage(hydrationData);
 
 	h2o.innerHTML = `
-		<p>All user's average water intake is ${avgOunces} ounces per day.</br></br>
-		Your water intake today was ${ouncesForDay} ounces</br></br>
+		<p>Today's all FitLit user's water intake averaged at ${avgOunces} ounces.</br></br>
+		Today you drank ${ouncesForDay} ounces of water.</br></br>
 		Your past week's water intake:</br></br>
```
Comment (st0nebreaker) [inline src/scripts.js:48]: changes a lot of sentences in this file to make more semantic sense

**Review by carlymclift — APPROVED** [review]
This looks great! thanks for changing a lot of the 'let' on 'const'

**Pull request merged**